tests/login/test_web.py

```diff
@@ -63,6 +65,22 @@ def test_auth_fail() -> None:
     assert res.status_code == 401
 
 
+@pytest.mark.unit_test
+def test_refresh() -> None:
```

Comment:
this test should use the refresh token with a invalid access token

Reply:
Yes, I didn't find a way to send refresh token during test. So I pushed like that, and come back later.